frontend/src/pages/destructionlist/hooks.ts

```diff
@@ -24,12 +24,16 @@ import { ExpandZaak, Zaak } from "../../types";
 /** The template used to format urls to an external application providing zaak details. */
 const REACT_APP_ZAAK_URL_TEMPLATE = process.env.REACT_APP_ZAAK_URL_TEMPLATE;
 
+interface PaginatedZakenResultsWithAction extends PaginatedZaken {
+  results: (Zaak & { action?: string })[];
+}
+
 /**
  * Hook that returns base props for most Zaak related DataGrid components.
  */
 export function useDataGridProps(
   storageKey: string,
-  paginatedResults: PaginatedZaken,
+  paginatedResults: PaginatedZakenResultsWithAction,
```

Comment:
I don't think swapping out `PaginatedZaken` for something else is the right approach here, can you explain what you are trying to do?